packages/xrpl/src/models/common/index.ts

```diff
@@ -20,6 +20,11 @@ export interface IssuedCurrencyAmount extends IssuedCurrency {
   value: string
 }
 
+export interface MPTAmount {
+  mpt_issuance_id: string
+  value: string
+}
+
 export type Amount = IssuedCurrencyAmount | string
```

Comment:
Could we Add MPTAmount here, since we group MPT and IOU typings in the isAmount check later on anyways? This way you would avoid the Amount | MPTAmount changes in a lot of the transactions

Reply:
Thanks for noting this! This choice was chosen intentionally (although I agree it's not the most ideal). The reason why MPTAmount isn't included because currently only a few transactions support MPTAmount (Payment, Clawback, etc), while other transactions (like CheckCreate) doesn't. This is why `Amount | MPTAmount` is seen for certain fields.

Reply:
We should probably create a ticket to remind to update it when MPTAmount gets incorporated into all the features

Reply:
Sure makes sense. Could you add an issue in the repo so we can track this for reference